shell/platform/darwin/macos/framework/Source/FlutterIOSurfaceWrapper.mm

```diff
@@ -0,0 +1,68 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterIOSurfaceWrapper.h"
+
+#include <OpenGL/gl.h>
+
+@interface FlutterIOSurfaceWrapper () {
+  IOSurfaceRef _ioSurface;
+}
+@end
+
+@implementation FlutterIOSurfaceWrapper
```

Comment:
maybe call this `FlutterIOSurfaceHolder`?

Reply:
Done